server/httpin/call.go

```diff
@@ -11,11 +11,29 @@ import (
 	"github.com/mattermost/mattermost-plugin-apps/utils/httputils"
 )
 
+// CallResponse contains everything the CallResponse struct contains, plus some additional
+// data for the client, such as information about the App's bot account.
+//
+// Apps will use the CallResponse struct to respond to a CallRequest, and the proxy will
+// decorate the response using the CallResponse to provide additional information.
+type CallResponse struct {
```

Comment:
Is it (too) difficult to isolate the `CallResponse` changes into its own PR?

Reply:
That is reasonably doable without too much effort: https://github.com/mattermost/mattermost-plugin-apps/pull/381